solvebio/test/test_shortcuts.py

```diff
@@ -59,45 +48,40 @@ def _validate_tmpl_fields(self, fields):
             elif f.name == 'aliases':
                 self.assertEqual(f.data_type, 'string')
 
-    @mock.patch('solvebio.resource.Depository.create',
-                side_effect=fake_depo_create)
-    @mock.patch('solvebio.resource.DepositoryVersion.create',
-                side_effect=fake_depo_version_create)
-    @mock.patch('solvebio.resource.Dataset.create',
-                side_effect=fake_dataset_create)
-    @mock.patch('solvebio.resource.DatasetTemplate.create',
-                side_effect=fake_data_tpl_create)
-    def test_create_dataset_upload_template(
-            self,
-            fake_depo_create,
-            fake_depo_version_create,
-            fake_dataset_create,
-            fake_data_tpl_create):
-        dataset_full_name = self.unique_dataset_name()
+    @mock.patch('solvebio.resource.Vault.all')
+    @mock.patch('solvebio.resource.Object.all')
+    @mock.patch('solvebio.resource.Dataset.create')
+    @mock.patch('solvebio.resource.DatasetTemplate.create')
+    def test_create_dataset_upload_template(self, TmplCreate,
+                                            DatasetCreate, ObjectAll,
+                                            VaultAll):
+        TmplCreate.side_effect = fake_dataset_tmpl_create
+        DatasetCreate.side_effect = fake_dataset_create
+        ObjectAll.side_effect = fake_object_all
+        VaultAll.side_effect = fake_vault_all
```

Comment:
I switched to putting the side effect stuff in the function since I think it reads easier.